src/lib/operators/jit_operator/operators/jit_abstract_sink.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "../../../storage/chunk.hpp"
+#include "../../../storage/table.hpp"
+#include "jit_abstract_operator.hpp"
+
+namespace opossum {
+
+/**
+ * JitAbstractSink is the last jit operator in the operator chain.
+ * It is responsible for:
+ * 1) adding column definitions to the output table
+ * 2) storing the tuples in output chunks
+ * 3) appending the output chunks to the output table
```

Comment:
This is detailed, but its most important high-level responsibility is to bring everything back into the data format the rest of the db works with